README.md

```diff
@@ -54,6 +54,13 @@ import IconComponent from './my-icon.svg?component'
 // <IconComponent />
 ```
 
+### Skip SVGO Optimizer for a file
+SVGO can be explicitly disabled for one file by adding the `?skipsvgo` suffix:
+```js
+import iconWithoutOptimizer from './my-icon.svg?skipsvgo'
+// '/assets/my-icon.2d8efhg.svg'
```

Comment:
By default, the return of this would be a component not a string. Could you update this comment to reflect that?

Reply:
fixed in 0c7e9dd9c05cea9ae4adbe710ee46a7f49380dcf